doc/releases/changelog-0.22.0.md

```diff
@@ -439,6 +491,12 @@
   display correctly in the docs.
   [(#2286)](https://github.com/PennyLaneAI/pennylane/pull/2286)
 
+* Docstring examples now display using the updated text-based circuit drawer.
+  [(#2252)](https://github.com/PennyLaneAI/pennylane/pull/2252)
+
+* Add docstring to `OrbitalRotation.grad_recipe`.
+  [(#2193)](https://github.com/PennyLaneAI/pennylane/pull/2193)
+
 <h3>Operator class refactor</h3>
```

Comment:
Hey @josh146, just double-checking if we want to have this dedicated section. If so, would it make sense to bring it further up?

Also, when checking this PR https://github.com/PennyLaneAI/pennylane/pull/2280, I came across the ability to pass a `custom_decomps` option to devices. This is not mentioned in the changelog as of now, is that intentional?

Reply:
Hey Antal, we definitely do not want this section here 😆 I can give it a go rewriting this section

>  I came across the ability to pass a custom_decomps option to devices.

Oh this was announced in a previous release :) https://pennylane.ai/blog/2021/12/pennylane-v020-released/#define-device-specific-custom-decompositions

Reply:
Oh, I see! Sure, thank you. Otherwise, I could distribute the items from there when doing a second pass on the PR.

> Oh this was announced in a previous release :) https://pennylane.ai/blog/2021/12/pennylane-v020-released/#define-device-specific-custom-decompositions

Got it! :+1: Thanks.